src/newshop.cpp

```diff
@@ -279,6 +281,10 @@ bool uninstall_ware_from_target_character(struct obj_data *obj, struct char_data
   } else {
     obj_from_cyberware(obj);
     GET_ESSHOLE(victim) += GET_CYBERWARE_ESSENCE_COST(obj);
+    if (GET_RACE(victim) >= RACE_GHOUL_HUMAN && GET_RACE(victim) <= RACE_GHOUL_TROLL)
+    GET_ESSHOLE(victim) += GET_CYBERWARE_ESSENCE_COST(obj) *= 1;
```

Comment:
`GET_CYBERWARE_ESSENCE_COST(obj) *= 1` means "the essence cost after multiplying by 1", so we don't need the `*= 1` component here. Same for below.

Reply:
Ah, gotcha, updated

Reply:
Since bioware essence value for the obj is doubled upon install for drakes, coupled it with a counterpart uninstall value setting of  x0.5 to handle the otherwise exponential bioware tumor growth issue